shim/src/unsafeRec.js

```diff
@@ -66,30 +66,23 @@ function createUnsafeRec(unsafeGlobal, allShims) {
   });
 }
 
-// todo: NEEDS COMMENT
-function sanitizeUnsafeRec(unsafeRec) {
-  // Ensures that neither the legacy accessors nor the function constructors
-  // can be used to escape the confinement of the evaluators to execute in the
-  // context.
-  repairFunctions(unsafeRec);
-  // we repair the accessors by injecting a shim string, so it gets the right types
-}
+const repairAccessorsShim = `"use strict"; (${repairAccessors})();`;
+const repairFunctionsShim = `"use strict"; (${repairFunctions})();`;
```

Comment:
Stringification doesn't need to be part of the shim itself. It's purely a concern of the unsafeRecord building process.